server/src/main/java/org/elasticsearch/action/support/replication/TransportReplicationAction.java

```diff
@@ -210,14 +197,14 @@ protected TransportReplicationAction(
             transportReplicaAction,
             executor,
             true,
-            true,
+            ArrayUtils.contains(flags, ActionFlags.BypassCircuitBreakerOnReplica) == false,
```

Comment:
The flags should represent deviations from the default so in this case, where we want to allow tripping the circuit breaker by default, we look for the absence of the bypass flag